azure-pipelines.yml

```diff
@@ -29,45 +29,43 @@ steps:
     TargetFolder: '$(Build.ArtifactStagingDirectory)/symbols'
     CleanTargetFolder: true
 
+- task: PublishSymbols@2
+  displayName: 'Enable Source Server'
+  inputs:
+    SymbolsFolder: '$(Build.ArtifactStagingDirectory)\symbols'
+    SearchPattern: '**/*.pdb'
+    SymbolServerType: TeamServices
+
 - task: PublishBuildArtifacts@1
   displayName: 'Upload Symbols Artifact'
   inputs:
     PathtoPublish: '$(Build.ArtifactStagingDirectory)/symbols'
     ArtifactName: Symbols
 
-- task: CopyPublishBuildArtifacts@1
+- task: PublishBuildArtifacts@1
   displayName: 'Upload Bin to drop'
   inputs:
-    Contents: |
-     src\MSBuildLocator\bin\**
-     
+    PathToPublish: '$(Build.StagingDirectory)\src\MSBuildLocator\bin'
     ArtifactName: bin
-    ArtifactType: Container
 
-- task: CopyPublishBuildArtifacts@1
+- task: PublishBuildArtifacts@1
   displayName: 'Upload NuGet to drop'
   inputs:
-    Contents: |
-     src\MSBuildLocator\bin\$(BuildConfiguration)\*.nupkg
-     
+    PathToPublish: '$(Build.StagingDirectory)\src\MSBuildLocator\bin'
     ArtifactName: pkg
-    ArtifactType: Container
 
-- task: CopyPublishBuildArtifacts@1
+- task: PublishPipelineArtifact@1
   displayName: 'Upload MicroBuildOutputs to drop'
   inputs:
-    CopyRoot: '$(Build.StagingDirectory)\MicroBuild\Output'
-    Contents: '**'
+    TargetPath: '$(Build.StagingDirectory)\bin'
     ArtifactName: MicroBuildOutputs
-    ArtifactType: Container
   continueOnError: true
 
-- task: CopyPublishBuildArtifacts@1
+- task: PublishBuildArtifacts@1
   displayName: 'Upload logs to drop'
   inputs:
-    Contents: '**\*log'
+    PathToPublish: '$(Build.StagingDirectory)\src\MSBuildLocator\bin'
```

Comment:
This doesn't look right to me?

Reply:
Copy-paste error 😅

That said, I'm not sure how to convert that one to the new style. The build with the penultimate commit worked, so we can go back to that if you want. I just wanted to also not use deprecated things if possible even though they weren't failing.